integrations/azure-devops/.port/resources/port-app-config.yaml

```diff
@@ -76,3 +89,15 @@ resources:
             changedDate: .fields."System.ChangedDate"
           relations:
             project: .__projectId | gsub(" "; "")
+            column: .__project.id + .fields."System.BoardColumn" | gsub(" "; "")
+  - kind: column
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .__project.id+.name | gsub(" "; "")
+          title: .name
```

Comment:
there can be repetition in the column names so this will make it hard to read which column/stage belongs to which board. id' have done boardName-columnName

Reply:
I still added project ID because board name is only unique in a project